etcdserver/etcdserverpb/rpc.proto

```diff
@@ -679,6 +679,9 @@ message LeaseGrantResponse {
 message LeaseRevokeRequest {
   // ID is the lease ID to revoke. When the ID is revoked, all associated keys will be deleted.
   int64 ID = 1;
+  // expire is the flag for indicating that the revoke request is invoked by expire.
+  // In such a case, permission checking can be skipped.
+  bool expire = 2;
```

Comment:
would it be cleaner to submit the expire request with root creds instead of having a flag in the rpc message?